scripts/checks.sh

```diff
@@ -25,8 +25,9 @@ external_fqdn="0.resinio.pool.ntp.org"
 low_mem_threshold=10 #%
 low_disk_threshold=10 #%
 wifi_threshold=40 #%, very handwavy
-
+expansion_threshold=80 #%
```

Comment:
@ZubairLK let me know if you think this is reasonable, or if I should increase it all the way to something like 95-98. In my testing most devices expand to 99.x%, but the ones that fail will expand to about ~25%.